tests/paramtests.py

```diff
@@ -501,6 +501,61 @@ def testPreprocessingTechniqueValueErrors(self):
         self.runValueErrorTest("pp_tech", [
             LPX.PP_NONE, LPX.PP_ROOT, LPX.PP_ALL])
 
+    def testSimpleRoundingHeuristic(self):
+        """Test the sr_heur parameter."""
+        if env.version<(4,57): return
+        legals=True, False
```

Comment:
Looking at 'src/lp.c', the argument `sr_heur` is declared to be an integer and not a boolean. Should 0, 1 be included in the list of legal values? Is there anything that can be checked after calling `integer()` with this argument to check if it was actually used?